### PR TITLE
Add Bitty and Beau's

### PR DESCRIPTION
**Pull request by risadams**
Added new coffee shop to data

A new coffee shop, Bitty & Beau's, has been added to the coffee_shops.json file. The addition includes details such as name, neighborhood, website, address and geographical coordinates. This does not include a photo, as I do not have my own photo  available and do not want to risk using a copyrighted image.

### CODE REVIEW

File: data/coffee_shops.json [hunk 1]
```diff
@@ -242,6 +242,21 @@
...
     },
+    {
+      "type": "Feature",
+      "properties": {
+        "name": "Bitty & Beau's",
+        "neighborhood": "Strip District",
+        "website": "https://www.bittyandbeauscoffee.com/location/pittsburgh",
+        "address": "1920 Smallman St Pittsburgh, PA 15222",
+        "roaster": "",
+        "photo": ""
```
Comment (Johngeorgesample) [inline data/coffee_shops.json:253]:
```suggestion
```

I've been meaning to fix it, but an empty `photo` property isn't supported right now. Would you mind removing it?

**Review by Johngeorgesample — CHANGES_REQUESTED** [review]
One small change but otherwise looks great! Thanks for opening the first PR!

**Comment by risadams** [comment]
N/P,  that's been updated.
I probably should have run locally first to test before assuming :)

File: data/coffee_shops.json [hunk 2]
```diff
@@ -242,6 +242,20 @@
...
       }
     },
+    {
+      "type": "Feature",
+      "properties": {
+        "name": "Bitty & Beau's",
+        "neighborhood": "Strip District",
+        "website": "https://www.bittyandbeauscoffee.com/location/pittsburgh",
+        "address": "1920 Smallman St Pittsburgh, PA 15222",
+        "roaster": "",
```
Comment (Johngeorgesample) [inline data/coffee_shops.json:252]:
```suggestion
        "roaster": ""
```

**Review by Johngeorgesample — CHANGES_REQUESTED** [review]
I should have caught this before. Happy to merge once this has been addressed!

**Pull request merged**